// //compile time error

// //sytax error

// console.log(1;

//runtime error
//Reference error

// try{

//     console.log("try block start here");
//     console.log(x);
//     console.log("try block ends here")

    

// }
//  catch(err) {

//     console.log("I AM INSIDE CATCH BLOCK")
//     console.log("Your error is here: ", err);

//  }
// finally{
//     console.log("I will run everytime, as i am finally block")
// }





let errorCode = 100;
if(errorCode == 100 ) {
    throw new Error("Invalid Json");
}